webapp/src/js/services/db-sync.js

```diff
@@ -157,7 +158,16 @@ angular
     const syncMeta = function() {
       const remote = DB({ meta: true, remote: true });
       const local = DB({ meta: true });
-      local.sync(remote);
+      local
+        .sync(remote, { filter: readOnlyFilter })
+        .on('change', changes => {
+          if (changes.direction === 'push' && changes.change && changes.change.docs) {
+            const deletes = changes.change.docs
+              .filter(doc => doc._id.startsWith('telemetry-') || doc._id.startsWith('feedback-'))
+              .map(doc => ({ _id: doc._id, _rev: doc._rev, _deleted: true, purged: true }));
+            return local.bulkDocs(deletes);
+          }
+        });
```

Comment:
My only concern here is that we won't clean up docs that were created before upgrading to 3.9.0. I'm wondering if we should record the current seq when we start and then, after the sync succeeds, purge everything up to that seq?

Reply:
That works! Esp because we can actually compare seqs in PouchDB :) 
I just pushed the test-less code, I'll get back to adding unit tests once the high prio tasks are taken care of.